libs/clients/license-client/README.md

```diff
@@ -0,0 +1,37 @@
+# License Client
+
+This library was generated with [Nx](https://nx.dev).
+
+## About
+
+This library serves as a license client factory for the following clients
+
+- Firearm license
+- Machine License
+- ADR License
+- Drivers License
+- Disability License
+
+There are two kinds of clients that can be generated that are meant to be used by different entities, _License Clients_ or _License Update Clients_.
+As of now, users can import either, or both, if they want, as long as they fulfill the dependancies. The clients have different roles, which are expanded upon below
+
+There is some doubt about the way the client types are separated, some thoughts below
+
+1. It feels weird to enable different methods based on if some config _(ids)_ is loaded or not. If it was just one big client some methods would only work if an ApiKey was supplied in config, and other would work if IDS config was provided.
```

Comment:
Isn't the main difference that some methods depend on user authentication (and require a User argument) while some require an admin scope/apikey to call the open api?

Then the main question is if we want to catch this at startup (because the app imports a nestjs module with missing configuration) or at runtime (if the app calls a method with missing configuration).

There's some extra boilerplate code involved with this split setup, but I think it is a good change.